session.go

```diff
@@ -248,6 +248,12 @@ func (s *Session) init() error {
 		return ErrNoConnectionsStarted
 	}
 
+	// Invoke KeyspaceChanged to let the policy cache the session keyspace
+	// parameters. This is used by tokenAwareHostPolicy to discover replicas.
+	if !s.cfg.disableControlConn && s.cfg.Keyspace != "" {
+		s.policy.KeyspaceChanged(KeyspaceUpdateEvent{Keyspace: s.cfg.Keyspace})
```

Comment:
can we just refresh all the keyspaces here?

Reply:
Well in the current impl [1] it's not needed, Query uses keyspace from session so effectively tokenAwareHostPolicy map can (and often does) store a single value. This is not too bad as applications most of the time use single keyspace or session per keyspace due to how you create a session. If you want we can:
* replace the meta map with a single value in tokenAwareHostPolicy
* remove the todo in [1]
* document that behavior, I'd appreciate some guidance where to put it except for [1] function doc    

[1] https://github.com/gocql/gocql/blob/master/session.go#L925